sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -81,36 +78,39 @@
           },
           "order": {
             "title": "Position",
-            "description": "Position of the file",
+            "description": "Position of the file, The lowest position means file is the main file.",
```

Comment:
The description seem not so easy to understand. Why not something like `the first file in the list is the main file`?

Reply:
OK, done!